src/resources/filters/crossref/preprocess.lua

```diff
@@ -110,5 +114,4 @@ function crossrefPreprocess()
   }
 end
 
-
-
+debug_5001 = 0
```

Comment:
Is this meant to be here?

Reply:
No, it's not. It slipped my print() checks because i needed to add some context around the printouts.